test/cmdlineTests/message_format_utf16/input.sol

```diff
@@ -1,3 +1,15 @@
 contract Foo {
```

Comment:
Should the test be called **message_format_utf8** instead of **...._utf16**?

Reply:
Hmm but the whole point is, that it is a more than 8-bit sequence.. maybe we just leave off the number completely.

Reply:
UTF-8 is an encoding scheme, it allows codepoints larger than 8-bits to be encoded in longer than 1 byte of data.

And since we do not support UTF-16 encoding (it is different to UTF-8), utf8 seems to be the proper naming choice.

Reply:
Renamed, 046f6e7.